device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-D108C8/config.bcm.j2

```diff
@@ -952,5 +952,3 @@ serdes_preemphasis_131=0x580c
 
 mmu_init_config="MSFT-TH2-Tier0"
 {{ IPinIP_sock }}
-
-phy_an_lt_msft=1
```

Comment:
why do you remove this attribute?

Reply:
fixed, mistakenly changed it when making the manual cherry-pick